sci-physics/formcalc/formcalc-9.9-r1.ebuild

```diff
@@ -44,7 +44,7 @@ src_install() {
 	doins -r "${S}"
 	# Copy executable, etc. permissions
 	for f in $(find * ! -type l); do
-		fperms --reference="${S}/$f" ${MMADIR}/${MY_P}/$f
+		fperms --reference="${S}/$f" "${D}"/${MMADIR}/${MY_P}/$f
 	done
```

Comment:
How about all of this `insinto`/`doins` and then restoring permissions, we just use `mv` or `cp`?